src/codegen_test.cpp

```
@@ -474,15 +474,15 @@ void CodeGen::emitMockCall(CallStmt &s) {
     const std::string &fnName = strExpr->value;
 
     // Check function exists
-    auto fit = functions_.find(fnName);
-    if (fit == functions_.end())
+    auto *fitOverloads = findFunction(fnName);
```

Clang (14.0.6)

[warning] variable 'fitOverloads' is not initialized

(cppcoreguidelines-init-variables)